ezyrb/rbf.py

```diff
@@ -47,9 +63,13 @@ def fit(self, points, values):
         """
         self.interpolators = []
         for value in values.T:
-            argument = np.hstack([points, value.reshape(-1, 1)]).T
             self.interpolators.append(
-                Rbf(*argument, smooth=self.smooth, function=self.kernel))
+                RBFInterpolator(points, value,
+                                neighbors=self.neighbors,
+                                smoothing=self.smooth,
+                                kernel=self.kernel,
+                                epsilon=self.epsilon,
+                                degree=self.degree))
```

Comment:
Are you sure we still need to loop over the components of `values`? The original documentation is not totally clear to me